README.md

```diff
@@ -264,7 +265,42 @@ car.make, car.model, price, color
 "Porsche", "9PA AF1", 60000, "green"
 ```
 
+### Example 7
 
+You can unwind arrays similar to MongoDB's $unwind operation using the `unwindPath` option.
+
+```javascript
+var json2csv = require('json2csv');
+var fs = require('fs');
+var fields = ['carModel', 'price', 'colors'];
+var myCars = [
+               { "carModel": "Audi",      "price": 0,      "colors": ["blue","green","yellow"] },
```

Comment:
Only thing that needs a fixup is the formatting on this array, and the extra spaces between fields.